src/plugins/lightbox/_base.scss

```diff
@@ -126,3 +126,10 @@ body {
 		@extend %global-display-block-important;
 	}
 }
+
+.mfp-wrap {
+	dialog{
+		background-color:transparent;
+		border:none;
+	 }
```

Comment:
Remove the spaces before the curly brace

Reply:
updated